livekit-plugins/livekit-plugins-aws/livekit/plugins/aws/log.py

```diff
@@ -1,3 +1,5 @@
 import logging
 
 logger = logging.getLogger("livekit.plugins.aws")
+for logger_name in ["botocore", "aiobotocore"]:
+    logging.getLogger(logger_name).setLevel(logging.INFO)
```

Comment:
We have a place in livekit-agents for those noisy loggers:

https://github.com/livekit/agents/blob/c6291d513527a1fdc34c20b4e5cb4be14d387333/livekit-agents/livekit/agents/cli/log.py#L15

Reply:
ah ok, i'll move over there